internal/cli/cloud_manager_clusters_list.go

```diff
@@ -23,36 +25,41 @@ import (
 	"github.com/spf13/cobra"
 )
 
-type cmClustersListOpts struct {
+type cloudManagerClustersListOpts struct {
 	*globalOpts
-	store store.AutomationGetter
+	store store.CloudManagerClustersLister
 }
 
-func (opts *cmClustersListOpts) init() error {
-	if opts.ProjectID() == "" {
-		return errMissingProjectID
-	}
-
+func (opts *cloudManagerClustersListOpts) init() error {
 	var err error
 	opts.store, err = store.New()
 	return err
 }
 
-func (opts *cmClustersListOpts) Run() error {
-	result, err := opts.store.GetAutomationConfig(opts.ProjectID())
+func (opts *cloudManagerClustersListOpts) Run() error {
+
+	var result interface{}
+	var err error
+
+	if opts.projectID == "" && config.Service() == config.OpsManagerService {
+		result, err = opts.store.ListAllClustersProjects()
+
+	} else {
+		var clusterConfigs *om.AutomationConfig
```

Comment:
[np] this could be a function but feel free to ignore unless you want to do other changes